src/JitInterface/src/CorInfoTypes.cs

```diff
@@ -137,9 +137,15 @@ public enum CorTokenType
         mdtBaseType = 0x72000000,
     }
 
-    public enum SIZE_T : ulong { } // Really should be IntPtr
-
-    public enum GSCookie : ulong { } // Really should be IntPtr
+    #if (ARM || ARMEL || X86)
```

Comment:
Also, the `SIZE_T*` on methods like getAddrModuleDomainID can be `void*` instead.